index.js

```diff
@@ -214,6 +227,53 @@ const lookUpCommand = (message) => {
     });
 };
 
+/*
+
+  const topFiveCommand = (message) => {};
+    The user uses this command to select up to 5 cards  from their inventory to display
+    any user can use the lookup command to look at another users top 5 cards, but only the owner of cards can select which cards go into their top 5
+    
+    To input a card to your top 5, the command must be entered as !showcase blue-eyes white dragon 
+
+
+
+*/
+
+const removeCardCommand = (message) => {
+  const cardToRemove = message.content.split(" ").slice(1).join(" ");
+  const ref = db.ref("users");
+  ref.once("value", function (snapshot) {
+    const userData = snapshot.val();
+    const userID = message.author.id;
+    const userInventory = userData[userID].inventory;
+    const inventoryRef = ref.child(userID).child("inventory");
```

Comment:
it's usually nicer to define variables closer to where they're used, like on line 257 below